sentry.properties

```diff
@@ -0,0 +1,5 @@
+defaults.url=https://sentry.io/
+defaults.org=browserstack-prod
+defaults.project=demo-app
+auth.token=b9d6c6b08b8b4211b8ef72f09b971c4f4e9b0b5d3fb84b129b9c64894185ceff
```

Comment:
Is the auth token supposed to be publicly available ?

Reply:
already updated